src/components/technologies/genericTechnologies/stor.py

```diff
@@ -498,12 +548,12 @@ def _define_stor_capex(self, b_tec, data):
 
         return b_tec
 
-    def write_results_tec_operation(self, h5_group, model_block):
+    def write_results_tec_operation(self, h5_group: h5py.Group, model_block: Block):
         """
         Function to report results of technologies after optimization
 
-        :param b_tec: technology model block
-        :return: dict results: holds results
+        :param Block b_tec: technology model block
+        :param h5py.Group h5_group: technology model block
```

Comment:
group explanation: group in HDF5 file of model run to which these results are saved.